src/iuse_actor.h

```diff
@@ -451,22 +420,31 @@ class inscribe_actor : public iuse_actor
 {
     public:
         // Can it write on items/terrain
-        bool on_items;
-        bool on_terrain;
+        bool on_items = true;
+        bool on_terrain = false;
 
         // Does it require target material to be from the whitelist?
-        bool material_restricted;
+        bool material_restricted = true;
 
         // Materials it can write on
-        std::vector<material_id> material_whitelist;
+        std::set<material_id> material_whitelist = {
+            material_id( "wood" ),
+            material_id( "plastic" ),
+            material_id( "glass" ),
+            material_id( "chitin" ),
+            material_id( "iron" ),
+            material_id( "steel" ),
+            material_id( "silver" )
+        };
 
         // How will the inscription be described
-        std::string verb; // "Write", "Carve"
-        std::string gerund; // "Written", "Carved"
+        std::string verb = "Carve";
+        std::string gerund = "Carved";
```

Comment:
Those two words are not extracted for translation anymore. But the translation is a mess for this iuse actor anyway (it translates a single word and forms a sentence with that word at various places, probably not easy to translate). I'll say it's acceptable for now.